cmd/protoc-gen-go-grpc/protoc-gen-go-grpc_test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -ex # Debugging enabled.
+
+trap "git clean --force --quiet" EXIT
+
+WD="$(dirname $0)"
+
+# Build protoc-gen-go-grpc binary and add to $PATH.
+pushd "${WD}"
+go build .
+PATH="${WD}:${PATH}"
+popd
+
+protoc \
+    --go-grpc_out=. \
```

Comment:
Can you make this output to a `mktemp` please (and `go build` to it too)?  Then you don't need the `git clean`.  If you do the `git clean` then you run the risk of blowing away all uncommitted changes which means you need to confirm the repo is pristine first and ...ugh.